tests/test_custodia.py

```diff
@@ -454,6 +454,21 @@ def assert_ssl_error_msg(self, msg, exc):
             return
         self.fail(str(exc))
 
+    def test_create_client_connection_error(self):
+        c = self.create_client('/secrets', None)
+        c.headers['REMOTE_USER'] = 'test'
+        self.assertRaises(ConnectionError, c.list_container, '/')
+
+    def test_create_client_connection_error_with_server_option(self):
+        # Create the client using the '--server <server>' option
+        # for Custodia servers
+        pass
+
+    def test_create_client_connection_error_with_uds_urlpath_option(self):
+        # Create the client using the '--uds_urlpath <uds_urlpath>' option
+        # for Unix Domain Sockets
+        pass
+
```

Comment:
cheimes, ask me to test against a server instance and a unix socket, but I don't know yet, how to create a client instance using this options properly, I just create this method to show my intention in implement it.